ydb/library/yql/dq/actors/spilling/compute_storage_actor.cpp

```diff
@@ -58,118 +60,92 @@ class TDqComputeStorageActor : public NActors::TActorBootstrapped<TDqComputeStor
         return this;
     }
 
-    NThreading::TFuture<IDqComputeStorageActor::TKey> Put(TRope&& blob) override {
-        InitializeIfNot();
-        // Use lock to prevent race when state is changed on event processing and on Put call
-        std::lock_guard lock(Mutex_);
-
-        FailOnError();
-
-        ui64 size = blob.size();
-
-        ActorSystem_->Send(SpillingActorId_, new TEvDqSpilling::TEvWrite(NextBlobId, std::move(blob)));
-
-        auto it = WritingBlobs_.emplace(NextBlobId, std::make_pair(size, NThreading::NewPromise<IDqComputeStorageActor::TKey>())).first;
-        WritingBlobsSize_ += size;
-
-        ++NextBlobId;
 
-        auto& promise = it->second.second;
+protected:
 
-        return promise.GetFuture();
+    void FailOnError() {
+        if (Error_) {
+            assert(false);
```

Comment:
Why do we need this assert here?

Reply:
We don’t need, I forgot to remove it after debugging
Thanks!